openapi-cli/src/test/resources/ballerina-to-openapi/data_type/tuple_types.bal

```diff
@@ -3,7 +3,7 @@ import ballerina/http;
 public type Pet record {
     int id;
     [int, string, decimal, float, User] address;
-    ReturnTypes? tuples;
```

Comment:
Shall we add this removing test for the tuple defined in the type definition with a nil value?

Reply:
Added via [6e54819](https://github.com/ballerina-platform/openapi-tools/pull/1596/commits/6e54819d1ee8e953dd351246c9cca9fd6f45bc0a)